src/Type/Definition/ResolveInfo.php

```diff
@@ -40,7 +40,7 @@ class ResolveInfo
      * Expected return type of the field being resolved.
      *
      * @api
-     * @var OutputType&Type
+     * @var Type
```

Comment:
Wasn't `OutputType&Type` more accurate?

Reply:
It gets a write from `$fieldDefinition->getType()`, which resolves to `Type`. I will handle this more gracefully in my follow-up PR.